Cython/Compiler/Parsing.py

```diff
@@ -3077,18 +3087,49 @@ def p_with_gil(s):
     else:
         return 0
 
-def p_exception_value_clause(s):
+def p_exception_value_clause(s, ctx):
+    """
+    Parse exception value clause.
+
+    Maps clauses to exc_check / exc_value as follows:
+     ______________________________________________________________________
+    |                             |            |             |             |
+    | Clause                      | exc_clause | exc_check   | exc_value   |
+    | ___________________________ | __________ | ___________ | ___________ |
+    |                             |            |             |             |
+    | <nothing> (default func.)   | False      | True        | None        |
+    | <nothing> (cdef extern)     | False      | False       | None        |
+    | noexcept                    | True       | False       | None        |
+    | except <val>                | True       | False       | <val>       |
+    | except? <val>               | True       | True        | <val>       |
```

Comment:
Just realized that `except *` is a thing as well. It's somewhat confusing that this isn't something like `except? *` (as otherwise, `except?` means `check=True` and `except` means `check=False`), but it's black-on-white in the [docs](https://cython.readthedocs.io/en/latest/src/tutorial/pure.html#static-typing).

```suggestion
    | except <val>                | True       | False       | <val>       |
    | except? <val>               | True       | True        | <val>       |
    | except *                    | True       | True        | None        |
```

Reply:
> as otherwise, `except?` means `check=True` and `except` means `check=False`

That horse has long left the barn, but it would be more intuitive to me (aside from jibing with `except *`) if `except` were to check for exceptions and `except?` ("?" -> more uncertainty...) didn't.